src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml

```diff
@@ -79,7 +79,8 @@
     <page-diagnostics-haltsystem>
         <name>Diagnostics: Halt system</name>
         <patterns>
-            <pattern>diag_halt.php*</pattern>
+            <pattern>/ui/core/halt*</pattern>
```

Comment:
it's either /ui/core/halt or /ui/core/halt/*

Reply:
first one when there is no /ui/core/halt/foo otherwise the second will catch both /ui/core/halt and /ui/core/halt/foo